<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="attack1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>attack1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attack2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>attack2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attack3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>attack3</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="myPic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="opPic">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="leaveButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Leave Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>520</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="opHealth">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>98</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>200/200</string>
    </property>
   </widget>
   <widget class="QLabel" name="myHealth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>98</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>200/200</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Press start to begin the battle...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>520</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="opPokemonName">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Opponent</string>
    </property>
   </widget>
   <widget class="QLabel" name="myPokemonName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Me</string>
    </property>
   </widget>
   <widget class="QLabel" name="opPokeCount">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>360</y>
      <width>98</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>[ O O O ]</string>
    </property>
   </widget>
   <widget class="QLabel" name="myPokeCount">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>98</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>[ O O O ]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
